CourseGrab/ViewControllers/Search/SearchTableViewController.swift

```diff
@@ -67,9 +67,13 @@ extension SearchTableViewController {
                 switch result {
                 case .value(let response):
                     DispatchQueue.main.async {
+                        guard let currentSearchText = self.textField.text,
+                              response.data.query == currentSearchText else {
```

Comment:
Small suggestion, but u can prob just do if response.data.query != self.textfield.text. Doesn't seem to me like ur using currentSearchText anywhere. The equality check should handle the null value case